main.go

```diff
@@ -289,13 +284,37 @@ func getCommands() []cli.Command {
 			Action:       token.AccessTokenCreateCmd,
 		},
 	}
-	allCommands := append(slices.Clone(cliNameSpaces), utils.GetPlugins()...)
-	allCommands = append(allCommands, scan.GetCommands()...)
+	allCommands := append(slices.Clone(cliNameSpaces), ConvertEmbeddedPlugin(securityCLI.GetJfrogCliSecurityApp())...)
+	allCommands = append(allCommands, utils.GetPlugins()...)
 	allCommands = append(allCommands, buildtools.GetCommands()...)
 	allCommands = append(allCommands, lifecycle.GetCommands()...)
 	return append(allCommands, buildtools.GetBuildToolsHelpCommands()...)
 }
 
+func ConvertEmbeddedPlugin(jfrogApp components.App) (embeddedCmd []cli.Command) {
+	// Convert commands
+	if converted, err := components.ConvertCommands(jfrogApp.Name, jfrogApp.Commands); err != nil {
+		clientlog.Error(fmt.Sprintf("failed adding '%s' embedded plugin as commands. Last error: %s", jfrogApp.Name, err.Error()))
```

Comment:
I think it is preferred this function returns errors, rather than adding error messages to the log.